services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -437,6 +444,107 @@ public void routeOptionsWithUTF8Chars_toUrlWithEncodedChars() {
     assertEquals(expectedEncodedUrl, url.toString());
   }
 
+  @Test
+  public void routeOptionWithExcludedPoint() {
+    RouteOptions options = RouteOptions.builder()
+        .coordinates("1.0,1.0;2.0,2.0")
+        .profile(DirectionsCriteria.PROFILE_DRIVING)
+        .excludePointsList(
+            Collections.singletonList(Point.fromLngLat(1.0, 2.0))
+        )
+        .build();
+    URL url = options.toUrl("testToken");
+    List<String> queryParameters = Arrays.asList(url.getQuery().split("&"));
+    assertTrue(
+        "url doesn't contain excluded point: " + url.toString(),
+        queryParameters.contains("exclude=point(1.0%202.0)")
+    );
+  }
+
+  @Test
+  public void routeOptionWithTwoExcludedPoints() {
+    RouteOptions options = RouteOptions.builder()
+        .coordinates("1.0,1.0;2.0,2.0")
+        .profile(DirectionsCriteria.PROFILE_DRIVING)
+        .excludePointsList(
+            Arrays.asList(
+                Point.fromLngLat(1.0, 2.0),
+                Point.fromLngLat(6.03, 8.07)
+            )
+        )
+        .build();
+    URL url = options.toUrl("testToken");
+    List<String> queryParameters = Arrays.asList(url.getQuery().split("&"));
+    assertTrue(
+        "url doesn't contain excluded point: " + url.toString(),
+        queryParameters.contains("exclude=point(1.0%202.0),point(6.03%208.07)")
+    );
+  }
+
+  @Test
+  public void routeOptionsWithExcludePointToFromUrl() throws MalformedURLException {
+    URL testUrl = new URL("https://api.mapbox.com/directions/v5/mapbox/driving-traffic/18.60289583391497,54.41121871390118;18.59400217318438,54.40983705017376.json?access_token=testToken&alternatives=true&annotations=distance%2Cduration%2Ccongestion%2Cspeed&geometries=geojson&language=en&overview=full&steps=true&exclude=point(18.595875353791087%2054.41000119108463)");
```

Comment:
Could you add additional test that has a URL with both exclusion types to verify we're parsing correctly?

Reply:
added